pkg/framework/test/democli/cmd/listPods.go

```diff
@@ -0,0 +1,65 @@
+// Copyright © 2017 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// listPodsCmd represents the listPods command
+var listPodsCmd = &cobra.Command{
+	Use:   "listPods",
+	Short: "List all pods",
+	Long:  `Give a list of all pods known by the system`,
+	Run: func(cmd *cobra.Command, args []string) {
+		runGetPods()
+	},
+}
+
+func runGetPods() {
+	config, _ := clientcmd.BuildConfigFromFlags("http://localhost:8080", "")
```

Comment:
Should this be a parameter instead?

Reply:
Definitely. We wont to have our framework generate some sort of config (the framework will be the thing that determines the port eg. the APIServer is listening on) and somehow expose that to the clients resp. the component under test. Until we have that and somewhat in the spirit of TDD we just did the simplest thing possible to make our stuff work. Next iterations will most definitely change that.